morecantile/models.py

```diff
@@ -60,6 +60,16 @@ class TileMatrixSet(BaseModel):
     boundingBox: Optional[BoundingBox]
     tileMatrix: List[TileMatrix]
 
+    @validator("tileMatrix")
+    def sort_tile_matrices(cls, v):
+        """Sort matrices by identifier"""
+        return sorted(v, key=lambda m: int(m.identifier))
+
+    def __iter__(self):
```

Comment:
@geospatial-jeff I think this is a nice feature that should be documented in the Readme?

Reply:
https://github.com/developmentseed/morecantile/pull/18/commits/0ce51df11ada548e8d529bb49c357877607e59fb